### DISCUSSION TITLE
No metadata - what am I missing?

### DISCUSSION THREAD

**Discussion by lloydsmart** [stashapp/stash#5192] [Q&A] [answered]
Hello, I have a bunch of content downloaded from Whisparr. It appears in Stash but there's no metadata attached such as performers, descriptions, studios, etc.

Metadata appears in Whisparr - is there a way to ship it over to Stash?

Everything just appears in "Scenes" and uses the filename as a title.

I'm used to Radarr and Sonarr, but I'm a noob with Whisparr and Stash - what am I doing wrong?

**Comment by DogmaDragon** [comment, marked as answer]
https://docs.stashapp.cc/beginner-guides/guide-to-scraping/

**Comment by lloydsmart** [comment]
Thank you - I'd done everything except click "Scrape All" 😂
